src/VisualStudio/IntegrationTest/TestUtilities/InProcess/InteractiveWindow_InProc.cs

```diff
@@ -218,11 +219,14 @@ public void InsertCode(string text)
         }
 
         public void WaitForLastReplOutput(string outputText)
-            => WaitForPredicate(GetLastReplOutput, outputText, s_contains, "contain");
+            => WaitForPredicate(GetLastReplOutput, outputText, s_equals, "is");
```

Comment:
📝 There is a chance this breaks another test, but if that happens I will correct the other test by changing it to use `WaitForLastReplOutputContains`.

Reply:
Some tests did break, and they have now been corrected.